Home Page

redpandarevolt opened the pull request:
Index.html 
Images - all copyright free
footer - copy and paste onto other pages
header - copy and paste onto other pages
media queries at 4 break points
Links to external sites are working
Link to internal pages need to be linked to correct .html page from other students.

redpandarevolt committed:
Home Page

Index.html
Images
footer
header
media queries at 4 break points

mduw reviewed (changes requested):
looks great to me overall. 
Suggest putting .jpg files into img folder.

isabelgiang commented on index.html:
```diff
@@ -0,0 +1,156 @@
+        <div class="top-nav">
+            <ul>
+                <li id="logo"><a href="T4G-Logo.png">
+                    <img src="T4G-Logo.png" alt="T4G logo" width="65" height="65">
```
Should these `width` and `height` attributes be moved out into the CSS file?

redpandarevolt replied:
CSS can be either in line on html page or in style sheet depending on how you want it to function.  The measurements were put directly on the image reference then added a width of 100% in CSS so they adjust with each devices width.   This is part of using the view-port option for device resizing.  One of many ways to skin the cat.  I'll look into it and see if there needs to be a modification.

isabelgiang commented:
Looks great to me 👍

redpandarevolt commented:
> looks great to me overall.
> Suggest putting .jpg files into img folder.

Mitchel that is a good idea for any image that is not linked.  If the image is a link will it cause he URL to be longer? As in it will turn into http://t4g-website/image/bike.jpg   vs  http://t4g-website/bike.jpg.  I'm not sure what is considered best practice on this.  

If it doesn't matter then I'm fine dumping all pictures into an /image folder. 
Good call!

redpandarevolt commented:
I also need to add commenting so future users have a good working idea of the moving parts and how to work with the site.

redpandarevolt committed:
Moved Images to folder

added img/ to all images directory

redpandarevolt committed:
Added pictures

Added student images to img directory

redpandarevolt committed:
Image adjustment

changes to apsect ratio of original images.

redpandarevolt committed:
Update index.html

Added Comments to index.html

redpandarevolt committed:
More changes

Added "About Us" to put info about Dr. G . & Dr. P
Added "About Us" to Nav bar
Added additional comments on index.html

redpandarevolt committed:
esthetic changes

-changed font-style and size in vaigation bar
-added rounded corner to header title to soften the look
-added gold hover to navigation links

redpandarevolt committed:
project links

added links from front page project tiles to the project page.

redpandarevolt committed:
Links Updated

Animation added to logo and project links

redpandarevolt committed:
Trial Font Changes

font changes to get a feel for look

redpandarevolt commented:
> looks great to me overall.
> Suggest putting .jpg files into img folder.

done!

redpandarevolt committed:
pull changes

isabelgiang commented:
@mduw can you approve the changes that @redpandarevolt has made on this PR? since you requested changes previously, merging is blocked until you approve those changes

isabelgiang commented:
> looks great to me overall.
> Suggest putting .jpg files into img folder.

this review specifically
